Source/Tests/Trakt.NET.Modules.Tests/TraktUsersModule/TraktUsersModule_GetPersonalListsStream_Tests.cs

```diff
@@ -0,0 +1,167 @@
+﻿namespace TraktNet.Modules.Tests.TraktUsersModule
+{
+    using FluentAssertions;
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Threading.Tasks;
+    using Trakt.NET.Tests.Utility;
+    using Trakt.NET.Tests.Utility.Traits;
+    using TraktNet.Enums;
+    using TraktNet.Exceptions;
+    using TraktNet.Objects.Get.Lists;
+    using TraktNet.Responses;
+    using TraktNet.Extensions;
+    using Xunit;
+
+    [TestCategory("Modules.Users")]
+    public partial class TraktUsersModule_Tests
+    {
+        private readonly string GET_PERSONAL_LIST_STREAM_URI = $"users/{USERNAME}/lists/{LIST_ID}";
+
+        [Fact]
+        public async Task Test_TraktUsersModule_GetPersonalListStreamAsync()
+        {
+            TraktMultipleUserListsQueryParams parameters = new TraktMultipleUserListsQueryParams
+            {
+                { USERNAME, LIST_ID },
+                { USERNAME, LIST_ID },
+            };
+            int totalLists = parameters.Count;
+            TraktClient client = TestUtility.GetMockClientForMultipleCalls(GET_PERSONAL_LIST_STREAM_URI, LIST_JSON, totalLists);
```

Comment:
The client is not setup correctly.
You need to add an extra method similar to `TestUtility.GetMockClientForMultipleCalls()`, but with OAuth, like in `TestUtility.GetOAuthMockClient()`.